server/paymentProviders/paypal/subscription.ts

```diff
@@ -344,8 +344,7 @@ const PayPalSubscription: PaymentProviderService = {
   },
 
   async refundTransaction(transaction, user, reason) {
-    // Subscription's transactions are always recorded with paypalSale
-    const captureId = transaction.data.paypalSale?.id;
+    const captureId = getCaptureIdFromPaypalTransaction(transaction);
```

Comment:
@znarf: Here I am trying to re-use our existing function that takes care of the `captureId` as it could be seen in the issue, https://github.com/opencollective/opencollective/issues/6234 the `captureId` isn't necessarily in `paypalSale` object.